onadata/settings/kobocat_settings.py

```diff
@@ -41,6 +41,8 @@
 else:
     TESTING_MODE = False
 
+MEDIA_URL= os.environ.get('KOBOCAT_MEDIA_URL', 'http://localhost:8001/media/')
```

Comment:
Alternatively could get the environment variable `KOBOCAT_URL` and append `/media/`, but this way seems more multi-host friendly.